core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -141,33 +136,15 @@ public void changeBreakpointState(final int lineNumber) {
      * Removes breakpoint mark.
      */
     private void deleteBreakpoint(final VirtualFile activeFile,
-                                  final Debugger debugger,
+                                  @Nullable final Debugger debugger,
```

Comment:
Annotations such @Nullable and javadocs are used for helping us understand how we can use some class or how it works without reading its body. No?